### ISSUE TITLE
1

### ISSUE THREAD

**Issue by 4z3rTy** [4z3rTy/wxcv#1]
<!DOCTYPE html>
<html>
	<head>
		<meta charset="utf-8">
		<meta name="viewport" content="width=device-width, initial-scale=1.0">
		<title></title>
		
		<link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet">
		<link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet">
		<link href="https://fonts.googleapis.com/css?family=Lato|Open+Sans|Raleway" rel="stylesheet">
		<link href="01.css" rel="stylesheet">
	</head>
	<style>
	img {
		display: block;
		margin: auto;
	}
	</style>
	<body>
		
		<nav class="navbar navbar-default" style="background-color:#0 ">
			<div class="container">
				<div class="navbar-header">
					<button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#myNavbar">
						<span class="icon-bar"></span>
						<span class="icon-bar"></span>
						<span class="icon-bar"></span>
					</button>
					<a class="navbar-brand" href="#">  Croq'& Share  </a>
				</div>
				<div class="collapse navbar-collapse" id="myNavbar">
					<ul class="nav navbar-nav">
						<li class="active"><a href="#">Home</a></li>
						<li><a href="#"> Toutes nos Recettes </a></li>
						<li><a href="#"> Actualités </a></li>
						<li><a href="#"> Rejoindre </a></li>
						<li><a href="#"> FAQ </a></li>
						<form class="form-inline my-2 my-lg-0">
						<input class="form-control mr-sm-2" type="text" placeholder="Miam !">
						<button class="btn btn-outline-success my-2 my-sm-0" type="submit">Search</button>
						</form>
					</ul>
					<ul class="nav navbar-nav navbar-right">
						<li><a href="#"><span class="glyphicon glyphicon-user"></span> Sign Up</a></li>
						<li><a href="#"><span class="glyphicon glyphicon-log-in"></span> Login</a></li>
					</ul>
				</div>
			</div>
		</nav>

		
		<div id="myCarousel" class="carousel slide" data-ride="carousel">

			
			<ol class="carousel-indicators">
				<li data-target="#myCarousel" data-slide-to="0" class="active"></li>
				<li data-target="#myCarousel" data-slide-to="1"></li>
				<li data-target="#myCarousel" data-slide-to="2"></li>
			</ol>

			
			<div class="carousel-inner">
				<div class="item active">
					<img src="http://www.gustiditalia.com/wp-content/uploads/skin-healthy-food.jpg" width="220" height="220">
				</div>
				<div class="item">
					<img src="https://i.pinimg.com/736x/c6/81/15/c68115d5e2b863f05ae1e51d10f4f036--healthy-mind-healthy-foods.jpg" width="220" height="220">
				</div>
				<div class="item">
					<img src="http://www.healthyfood.co.uk/wp-content/uploads/2015/06/healthy-diabetes-diet.jpg" width="220" height="220">
				</div>
			</div>

			
			<a class="left carousel-control" href="#myCarousel" data-slide="prev">
				<span class="glyphicon glyphicon-chevron-left"></span>
				<span class="sr-only">Previous</span>
			</a>
			<a class="right carousel-control" href="#myCarousel" data-slide="next">
				<span class="glyphicon glyphicon-chevron-right"></span>
				<span class="sr-only">Next</span>
			</a>
		</div>

		
		<div class="container">
			<div class="row text-center">
				<div class="col-sm-4">
					<img height="120" src="https://upload.wikimedia.org/wikipedia/commons/thumb/2/25/Bol_d%27aligot.jpg/1200px-Bol_d%27aligot.jpg">
					<h1>Aligot</h1>
					<button class="btn btn-primary">Decouvrir</button>
				</div>
				<div class="col-sm-4">
					<img height="120" src="https://upload.wikimedia.org/wikipedia/commons/thumb/6/6b/Saint-Nectaire.jpg/1200px-Saint-Nectaire.jpg">
					<h1>Saint-Nectaire</h1>
					<button class="btn btn-primary">Decouvrir</button>
				</div>
				<div class="col-sm-4">
					<img height="120" src="https://www.quizz.biz/uploads/quizz/59834/10_6Bmyw.jpg">
					<h1>Tarte aux pralines</h1>
					<button class="btn btn-primary">Decouvrir</button>
				</div>
			</div>
			<hr>
			<div class="row">
				<div class="col-sm-4 col-sm-offset-4">
					<h1> Evaluer nous </h1>
					<div class="form-group">
						<input class="form-control" type="text" placeholder="Username">
					</div>
					<div class="form-group">
						<input class="form-control" type="password" placeholder="Position">
					</div>
					<div class="form-group">
						<label class="checkbox-inline"><input type="radio" name="myRadio">Top</label>
						<label class="checkbox-inline"><input type="radio" name="myRadio">Meh</label>
						<label class="checkbox-inline"><input type="radio" name="myRadio">Sucks</label>
					</div>
					<div class="form-group">
						<textarea class="form-control" rows="8"></textarea>
					</div>
					<div class="form-group text-center">
						<button class="btn btn-primary">Submit</button>
					</div>
				</div>
			</div>
			<hr>
			<div class="text-muted text-center form-group">
				© 2017 Croq'& Share, Inc.
			</div>
		</div>

		
		<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
		<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
	</body>
</html>
